pelicun/uq.py

```diff
@@ -856,10 +841,10 @@ def fit_distribution_to_sample(
 
     # There is nothing to gain from a time-consuming optimization if..
     #     the number of samples is too small
-    if (n_samples < 3) or (
+    small_n_samples = 3
+    if (n_samples < small_n_samples) or (
```

Comment:
Why do we have to create variables for numbers that used once in the code? This `small_n_samples` is an example, but I see it happening many times with this update. I think it makes the code harder to read and I don't see the advantage. Please point me to some resource that explain why this is a good idea.

Reply:
These changes are driven by [PLR2004](https://docs.astral.sh/ruff/rules/magic-value-comparison/). Having a verbal description (in the form of the variable name) instead of a number inside comparisons is recommended to help make the code speak for itself. What do you think? We don't necessarily need to conform to the rule, we can ignore it in our config.

Reply:
I think the example behind that link has the `>=` mixed up with `<=`, but I see their point nevertheless. This is related to the discussion we had about commenting: If we use better variable names, there is no need for many of the comments. I agree with that and I would like to support this approach. But, if this is what we want, then, I'm sorry, we need better variable names than `small_n_samples`. 
`3` is the minimum sample size for optimization to make sense here. For a sample of 1 or 2 realizations, the method of moments will work just fine. So, I would say `min_sample_size_for_optimization` if I want this to replace a comment and be descriptive.

Reply:
Excellent, I agree. b201aafa